code/game/turfs/open/space/space.dm

```diff
@@ -31,49 +31,36 @@
 /**
  * Space Initialize
  *
- * Doesn't call parent, see [/atom/proc/Initialize]
+ * Doesn't call parent, see [/atom/proc/Initialize].
+ * When adding new stuff to /atom/Initialize, /turf/Initialize, etc
+ * don't just add it here unless space actually needs it.
  */
 /turf/open/space/Initialize(mapload)
 	SHOULD_CALL_PARENT(FALSE)
-	icon_state = SPACE_ICON_STATE
+	icon_state = SPACE_ICON_STATE(x, y, z)
 	air = space_gas
-	vis_contents.Cut() //removes inherited overlays
-	visibilityChanged()
```

Comment:
These two were added by https://github.com/tgstation/tgstation/pull/44787.

This was back when everything was being moved to vis_contents because we thought it was faster, but it wasn't.

`visibilityChanged()` specifically was important when AI static was vis_contents, apparently.

I tested if the bugs these meant to fix were still around, and the answer is no.

Reply:
what does this do to openspace

Reply:
vis_contents is also cut in Destroy() so i dont think this will do anything

Reply:
oh it is? this is safe then yeah, could be removed from base turfs.
Not sure about `visibilityChanged` honestly. it should be a safe removal because space tiles will never block opacity,

Should ABSOLUTELY be documented tho, fragile.